### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by we11done**
Created
- LandingPage.js
- LandingPage_style.css
- Experience.js
- Experience_style.css

LandingPage.js
- Rendering headers and image of the landing page.
- Arguable alignment of the header, Sehyun and I had a conversation about this and we could not decide whether I should use center alignment or leave it as left alignment.

Experience.js
- Rendering images and information of the past careers.
- Sehyun and I decided to align the texts as Sehyun portraited in his project, but we also would like to know your suggestions.

Minor Edits
- Font (From Uber Move to Rubik)
- Mobile Menu button color (From violet to white)

Images Added
- Flybits.png
- Klick.png
- LandingPage.png
- Universe_FullStack.png
- Universe.png
- Verticalscope.png

Images for Experience.js
- Sehyun and I decided to resize all the images to make a perfect alignment. But if you have any better suggestions or doubts about the alignment, please let me know.

### CODE REVIEW

File: src/components/Experience.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            Large scale cloud infrastructure architecture 
+            and development. Exploring diverse fields of 
+            platform and centralizing a core infrastructure 
+            for all platforms on GCP.
+          </p>
+          <button>Learn More</button>
+        </div>
+      </div>
+
+    <div class="Container">
```
[the remark this answers is not in the document]
Comment (we11done) [reply src/components/Experience.js:27]: I do not see any indentation miss, where is it?

**Comment by we11done** [comment]
### Landing Page
Main Updates
- Images changed
- Minor position adjustment on the Experience section in tablet width(768px)

Designer request
- Position of text-nav elements adjusted to the right
- Minor position adjustment on the footer elements
- Nav social media icon updated to square icons from official icons

File: src/components/Navbar.js [hunk 2]
```diff
@@ -25,48 +25,49 @@ export const Navbar = () => {
...
-          </li>
-          <li>
-            <Link to="https://www.instagram.com">
-              <FontAwesomeIcon icon={faInstagram} />
-            </Link>
-          </li>
-        </ul>
-      </div>
-    </nav >
+        <div className="nav">
```
Comment (youngjk) [inline src/components/Navbar.js:38]: Indentation

File: src/components/Footer.js [hunk 3]
```diff
@@ -6,63 +6,67 @@ import { faLinkedinIn, faFacebookF, faInstagram } from "@fortawesome/free-brands
...
-          </li>
-          <li>
-            <Link to="/career">Career</Link>
-          </li>
-          <li>
-            <Link to="/investment">Investment</Link>
-          </li>
-        </ul>
-      </div>
+        <div className="footer-wrap">
```
Comment (youngjk) [inline src/components/Footer.js:15]: Indentation x2

File: src/components/TitleImg.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          <h2>
+            Young Jin Kim
+          </h2>
+          <h1>
+            PLATFORM
+          </h1>
+          <h1>
+            ENGINEER
+          </h1>
+
```
Comment (youngjk) [inline src/components/TitleImg.js:18]: spacing

File: src/components/Experience.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+            Large scale cloud infrastructure architecture 
+            and development. Exploring diverse fields of 
+            platform and centralizing a core infrastructure 
+            for all platforms on GCP.
+          </p>
+          <button>Learn More</button>
+        </div>
+      </div>
+
+    <div className="container" id="reverse">
```
Comment (youngjk) [inline src/components/Experience.js:27]: identation

**Review by youngjk — APPROVED** [review]
Early approval
- Check indentation/spacing (Consistence)

**Pull request merged**